microsoft-edge/devtools-guide.md

```diff
@@ -94,4 +98,42 @@ These shortcuts control the main DevTools window and/or work across all tools.
 | Find in search box | `Ctrl`+`F` |  
 | Give focus to console at the bottom | `Alt`+`Shift`+`I` |  
 | Launch DevTools to Console | `Ctrl`+`Shift`+`J` |  
-| Refresh the page.  **NOTE**:  If you're debugging and paused at a breakpoint, this resumes execution first.  | `Ctrl`+`Shift`+`F5`, `Ctrl`+`R` |  
+| Refresh the page | `Ctrl`+`Shift`+`F5`, `Ctrl`+`R` |  
+
+> [!NOTE]
+> If you're debugging and paused at a breakpoint, the **Refresh the page** action resumes the runtime first.
+
+<!-- image links  -->  
+
+[ImageDevtoolsEdgehtml]: /microsoft-edge/devtools-guide/media/devtools.png "Microsoft Edge (EdgeHTML) DevTools"  
+[ImageDevtoolsGuideEdgehtmlChooselocal]: /microsoft-edge/devtools-guide/media/chooser_local.png "DevTools app Local panel"  
```

Comment:
Do we need to use "app"?

Reply:
This is old EdgeHTML content, so few changes have been made.